wire-ios-sync-engine/Source/Calling/WireCallCenterV3+Events.swift

```diff
@@ -507,28 +507,108 @@ extension WireCallCenterV3 {
         conversationId: AVSIdentifier,
         completion: @escaping (_ clients: String) -> Void
     ) {
-        handleEventInContext("request-clients") { [weak self, encoder] _ in
+        handleEventInContext("request-clients") { [weak self, encoder] context in
+            guard let self else { return }
+
+            // Check if this is an MLS conference call
+            if conversationType(from: conversationId) == .mlsConference {
+                guard
+                    let snapshot = callSnapshots[conversationId],
+                    let groupIDs = snapshot.groupIDs
+                else {
+                    Self.logger.error("Cannot get group IDs for MLS conference")
```

Comment:
suggestion: add the conversationId to the log:

```suggestion
                    Self.logger.error("Cannot get group IDs for MLS conference", attributes: [.conversationId: conversationId.safeForLogging)
```